src/Tribe/Linked_Posts.php

```diff
@@ -934,14 +934,12 @@ public function saved_linked_post_dropdown( $post_type, $current = null ) {
 				type="hidden"
 				class="tribe-dropdown linked-post-dropdown"
 				name="' . esc_attr( $name ) . '"
-				id="saved_' . esc_attr( $post_type ) . '"
-				data-placeholder="' . $placeholder . '"
-				data-search-placeholder="' . $search_placeholder . '" ' .
+				id="saved_' . esc_attr( $post_type ) . '" data-placeholder="' . $placeholder . 
+				'" data-search-placeholder="' . $search_placeholder . '" ' .
```

Comment:
Hey there @MZAWeb - this places the closing quote for the `data-placeholder` attribute on a different line ... is that intentional ... seems like maybe it isn't based on the PR description?

Reply:
No, that's definitely a mistake. Not sure what happened. The one below is the fix.

Sorry about that.